Scripts/GenerateSampleViewSourceCode.swift

```diff
@@ -116,7 +118,7 @@ private let sampleStructs = sampleMetadata
             var category: String { \"\(sample.category)\" }
             var description: String { \"\(sample.description)\" }
             var snippets: [String] { \(sample.snippets) }
-            var tags: Set<String> { \(sample.keywords) }
+            var tags: Set<String> { \(Set(sample.keywords).symmetricDifference(Set(sample.relevantApis))) }
```

Comment:
This should be `subtracting` because keywords was constructed from (tags + APIs). See https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/b9d5c704d76f862f3404e2cccaf1f67a81d1afc6/Scripts/CI/common.py#L343